### ISSUE TITLE
Confirm remove role has undescriptive text

### ISSUE THREAD

**Issue by David-Strupinski** [uw-it-aca/uw-mailman3-web#49]
implementation issue on "Confirm remove role" page in Postorius

Was done by changing trans tags; cannot do that
